generic/samples/native-client/src/commonMain/kotlin/runner.kt

```diff
@@ -0,0 +1,16 @@
+import io.ktor.client.*
+import io.ktor.client.engine.*
+import io.ktor.client.request.*
+
+suspend fun runClient(engine: HttpClientEngine) {
+    val client = HttpClient(engine)
+    try {
+        val response = client.get<String>("http://example.com")
+        print(response)
+    }
+    finally {
+        // To prevent "IllegalStateException: Cannot execute task because event loop was shut down"
```

Comment:
I wonder if this is clear for people to understand why. Is it a Ktor bug or is it required, etc.

Reply:
@e5l, is it a problem Ktor - native integration problem?

Reply:
The issue about that error [KTOR-1071](https://youtrack.jetbrains.com/issue/KTOR-1071)

Reply:
Then I think it's important that this is highlighted in the demo sample and then updated when the issue is resolved. A reference to the link would be good.